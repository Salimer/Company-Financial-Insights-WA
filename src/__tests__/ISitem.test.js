import React from 'react';
  const index = 0;

  const { container } = render(<ISitem parameter={parameter} index={index} />);
  expect(container.firstChild).toMatchSnapshot();
});
