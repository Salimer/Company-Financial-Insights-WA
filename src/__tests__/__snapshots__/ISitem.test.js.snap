// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the income statement item correctly 1`] = `
<li
  class="flex justify-between text-white h-full p-5 bg-buttonSecondColor"
>
  <span
    class=""
  >
    Revenue
  </span>
  <span
    class=""
  >
    1000000
  </span>
</li>
`;
